pyblish_lite/app.py

```diff
@@ -68,7 +68,7 @@ def show(parent=None):
 
         self._window.show()
         self._window.activateWindow()
-        self._window.resize(430, 600)
+        self._window.resize(settings.WindowSize[0], settings.WindowSize[0])
```

Comment:
Ops, typo. `[0]` in both arguments.

How about this?

```python
self._window.resize(*settings.WindowSize)
```

It's expand the argument automatically.